src/plugins/sorting-task/SortingTaskNode/EditableTerms/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { ITerm, ITermsList } from "../interfaces";
+import EditableTermsItem from "./EditableTermsItem";
+
+interface IProps {
+  terms: ITermsList,
+  onEdit: (updatedTerms: ITermsList) => void,
+}
+
+export default class EditableTerms extends React.PureComponent<IProps> {
+
+  public render() {
+    const rows = this.getRows().map((term, index) => {
+      return (
+        <EditableTermsItem
+          key={index}
+          term={term}
+          onEdit={this.editHandler(index)}
+        />
+      );
+    });
+
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Begriff</th>
+            <th>Erklärung</th>
+          </tr>
+        </thead>
+        <tbody>
+          {rows}
+        </tbody>
+      </table>
+    );
+  }
+
+  /*
+   * This method returns a list of terms to render as rows.
+   * If the last row is not empty, it appends an empty row.
+   * This way, users are able to add a new term to the list
+   * without the need to manuall append a row.
+   */
+  protected getRows(): ITermsList {
```

Comment:
Adding a new `term` without requiring an additional button click is a good idea I think. Im not sure if its intuitiv for the average user but thats something we should test and discuss with Konstantin.